assessments/projects/openfga/joint-assessment.md

```diff
@@ -0,0 +1,647 @@
+- Implement a singleflight CheckResolver to avoid concurrency evaluation of overlapping subproblems. [#1301](https://github.com/openfga/openfga/issues/1301)
+
+## Security Issue Resolution
+
+### Responsible Disclosure
+
+OpenFGA vulnerability management is described in the official project security documentation [SECURITY.md](https://github.com/openfga/.github/blob/main/SECURITY.md).
+
+### Incident Response
+
+The OpenFGA maintainers bear the responsibility of monitoring and addressing reported vulnerabilities. Identified issues undergo prioritized triage, with immediate escalation upon confirmation. The triage process is conducted in private channels.
+
+Adhering to the GitHub security advisory process, OpenFGA initiates the CVE (Common Vulnerabilities and Exposures) request upon issue identification. The resolution is developed in a private branch associated with the CVE.
+
+Upon confirmation of the fix's effectiveness, it is released through a new patch for each major supported version of OpenFGA.
+
+The changelog will link to the CVE, which will describe the vulnerability and its mitigation. Any public announcements sent for these fixes will be linked to [the release notes](https://github.com/openfga/openfga/releases/tag/v1.3.2).
+
+All OpenFGA security issues can be found on the [Github advisories page](https://github.com/openfga/openfga/security/advisories).
+
+### Closed security issues and vulnerabilities
+
+At the time of the joint assessment, OpenFGA did not have any closed or fixed 
+issues in their repo with the label "security".
+
+## Hands-on assessment
+
+The hands-on assessment is a lightweight review of the project's internal security as
+well as the current recommendation configuration, deployment, and interaction
+with regard to security.  Hands-on assessments are subject to security reviewer
+availability and expertise.  They are not intended to serve as an audit or
+formal assessment and are no guarantee of the actual security of the project.
+
+**OpenFGA did/did not receive a hands-assessment from TAG-Security.**
+
+<!-- Commented out in case a hands-on assessment is requested and performed
+*If a hands-on assessment was performed, the below format should be used for
+reporting details*
+
+| | |
+| -- | -- |
+| Date of assessment | mmddyyyy-mmddyyyy |
+| Hands-on reviewers | name, github handle |
+
+| Finding Number | Finding name | Finding Notes | Reviewer |
+| -- | -- | -- | -- |
+| | | |
+
+### Hands-on assessment result
+
+General comments and summary of the hands-on assessment with any recommendations
+worth noting.  If nothing found use the below example:
+
+> TAG-Security's hands-on assessment did not reveal any significant or notable
+security findings for OpenFGA. This outcome does not indicate that none exist,
+rather that none were discovered.
+-->
+
+## Roadmap [TODO] fill in consultation with project team
+
+* Project Next Steps. Link to your general roadmap, if available, then list
```

Comment:
Created a specific project for tracking actions items here https://github.com/orgs/openfga/projects/8

Reply:
Have added this project under 'Next Steps" section.